home-exercises/src/main/java/advancedtest/exercise5/App.java

```diff
@@ -0,0 +1,20 @@
+package advancedtest.exercise5;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Function;
+
+public class App<T> {
+    public  void main(String[] args) {
+        List<Integer> integerList = Arrays.asList(1, 2, 3, 4, 5, 6);
+        double percent = partOf(integerList, (Integer n) -> n % 2 == 1;
```

Comment:
Man izskatās, ka šis vispār nekompilējas.

Reply:
Nepaspēju līdz galam izpildīt testa laikā, tagad visam jābūt korekti